libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/Base.hs

```diff
@@ -775,7 +760,8 @@ instance
       transWithdrawals ws = fmap Agda.MkHSMap . forM (Map.toList ws) $
         \(cred, Coin amount) -> do
           agdaCred <- toSpecRep cred
-          pure (Agda.RwdAddr () agdaCred, amount)
+          network <- toSpecRep Testnet -- TODO where should this really come from?
```

Comment:
So changing the generator to generate Testnet only as you did, didn't help with this?

Reply:
It works, but I would expect this `network` field to be present somewhere in the withdrawal. For some reason the spec uses `RwdAddr` which has a network field, but in the implementation we use a `Credential`, which doesn't have a network field.